test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -916,6 +931,10 @@ UCS_TEST_P(test_ucp_sockaddr, compare_cm_and_wireup_configs) {
     ucp_worker_cfg_index_t cm_ep_cfg_index, wireup_ep_cfg_index;
     ucp_ep_config_key_t *cm_ep_cfg_key, *wireup_ep_cfg_key;
 
+    if (!cm_use_all_devices()) {
```

Comment:
common helper?

Reply:
better to move to `UCS_TEST_SKIP_COND_P`

Reply:
good catch, done